app/controllers/users_controller.rb

```diff
@@ -34,6 +36,18 @@ def judge
 
   private
 
+  def css_id
+    @css_id ||= valid_css_id_or_nil
```

Comment:
Suggestion below could leave this shortened to
```suggestion
    @css_id ||= params[:css_id].presence && to_valid_css_id(params[:css_id])
```